lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java

```diff
@@ -25,12 +25,13 @@ public S3ObjectCopier(
 
   public boolean copyS3ObjectToNewLocation(final S3Object s3Object) {
     try {
+      LOGGER.info("Copying {} to {}", s3Object.toString(), destinationBucket);
       final String destinationKey = destinationKeyPrefix + getFileNameFromKey(s3Object.getKey());
       amazonS3Client.copyObject(
           s3Object.getBucket(), s3Object.getKey(), destinationBucket, destinationKey);
       return true;
     } catch (AmazonServiceException e) {
-      LOGGER.error("Unable to copy file", e);
+      LOGGER.error("Unable to copy file '" + s3Object.toString() + "' to " + destinationBucket, e);
```

Comment:
Shouldn't need an explicit `toString()`

```suggestion
      LOGGER.error("Unable to copy file '" + s3Object + "' to " + destinationBucket, e);
```

Reply:
done